docs/presets/pastel-rainbow.md

```diff
@@ -0,0 +1,21 @@
+[Return to Presets](./README.md#gruvbox-rainbow)
+
+# Pastel Rainbow Preset
+
+This preset is the same as [Gruvbox Rainbow](./gruvbox-rainbow.md) but with the colors of [Pastel Powerline](./pastel-powerline.md).
+
+![Screenshot of Pastel Rainbow preset](/presets/img/pastel-rainbow.png)
+
+### Prerequisites
+
+- A [Nerd Font](https://www.nerdfonts.com/) installed and enabled in your terminal
+
+### Configuration
+
+```sh
+starship preset pastel-rainbow -o ~/.config/starship.toml
+```
+
+[Click to download TOML](/presets/toml/pastel-rainbow.toml)
+
+<<< @/.vuepress/public/presets/toml/gruvbox-rainbow.toml
```

Comment:
change the path <<< @/public/presets/toml/pastel-rainbow.toml

Reply:
Thanks, I have edited both errors.